addons/true-youtube-links/userscript.js

```diff
@@ -1,11 +1,14 @@
-﻿export default async function ({ addon, global, console }) {
+﻿export default async function ({ addon, console }) {
```

Comment:
I think this will break. Not sure though.
```suggestion
﻿export default async function ({ addon, global, console }) {
```

Reply:
It is okay to remove unused variables declared within a function.

Reply:
Yes, but `console` will now be `global`.

Reply:
No, it won't.

Reply:
That's not how [destructuring](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment) works.